{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:185:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:35.394Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:185:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:35.395Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:185:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:35.396Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:[object Object] failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:189:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:35.402Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:[object Object] failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:189:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:35.403Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:185:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:35.657Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:185:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:35.658Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:185:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:35.659Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:[object Object] failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:189:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:35.663Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:[object Object] failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:189:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:35.664Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:185:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:36.100Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:185:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:36.101Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:185:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:36.102Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:[object Object] failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:189:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:36.103Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:[object Object] failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:189:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:36.104Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:185:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:36.843Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:185:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:36.845Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:185:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:36.845Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:[object Object] failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:189:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:36.847Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:[object Object] failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:189:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:36.848Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:185:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:38.098Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:185:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:38.099Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:185:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:38.100Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:[object Object] failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:189:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:38.102Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:[object Object] failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:189:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:38.103Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:185:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:40.232Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:185:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:40.234Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:185:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:40.234Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:[object Object] failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:189:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:40.236Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:[object Object] failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:189:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:40.237Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:185:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:43.853Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:185:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:43.854Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:185:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:43.855Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:[object Object] failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:189:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:43.856Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":2856,"level":50,"as1":true,"msg":"[worker] Uncaught exception, stack trace: Error: Redis connection to 127.0.0.1:[object Object] failed - connect ECONNREFUSED 127.0.0.1:6379\n    at RedisClient.on_error (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:189:24)\n    at Socket.<anonymous> (/Users/rumpus/Webstorm/as1/node_modules/primus-redis-rooms/node_modules/redis/index.js:95:14)\n    at emitOne (events.js:116:13)\n    at Socket.emit (events.js:211:7)\n    at emitErrorNT (internal/streams/destroy.js:64:8)\n    at _combinedTickCallback (internal/process/next_tick.js:138:11)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","time":"2018-05-03T01:19:43.857Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":7539,"level":50,"loginuser":true,"msg":"[login] General exception... Invalid login...","time":"2018-05-03T01:52:00.847Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":7539,"level":50,"posgetproduct":true,"msg":"[posgetproduct] General exception... column p1.barcode does not exist","time":"2018-05-03T01:52:09.009Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":9315,"level":50,"loginuser":true,"msg":"[login] General exception... Invalid login...","time":"2018-05-03T02:12:28.934Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":11913,"level":50,"loginuser":true,"msg":"[login] General exception... Invalid login...","time":"2018-05-03T02:16:44.817Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":16746,"level":50,"posgetproduct":true,"msg":"[posgetproduct] General exception... column p1.barcode does not exist","time":"2018-05-03T02:54:54.544Z","v":0}
{"name":"as1","hostname":"Mitchams-iMac.local","pid":18010,"level":50,"loginuser":true,"msg":"[login] General exception... Invalid login...","time":"2018-05-03T03:15:15.164Z","v":0}
{"name":"as1","hostname":"Ians-MacBook-Pro.local","pid":14252,"level":50,"poscashsale":true,"msg":"[poscashsale] General exception... Cannot read property 'posclientid' of undefined","time":"2018-05-03T16:01:17.091Z","v":0}
